### PR TITLE
feat: create return reason

### PR DESCRIPTION
**Pull request by christiananese**
Closes CC-305

### CODE REVIEW

File: packages/admin-next/dashboard/src/components/table/data-table/data-table-root/data-table-root.tsx [hunk 1]
```diff
@@ -52,6 +52,10 @@ export interface DataTableRootProps<TData> {
    * Whether the table is empty due to no results from the active query
    */
   noResults?: boolean
+  /**
+   * Whether to display the tables header
```
Comment (christiananese) [inline packages/admin-next/dashboard/src/components/table/data-table/data-table-root/data-table-root.tsx:56]: TBH I'm not a big fan of "negative" naming for props, but wanted to stay consistent with other props in this component

**Comment by olivermrbl** [comment]
We might want to have a max-width on the column and truncate the text – wdyt?

CleanShot 2024-08-08 at 21 12 37@2x

**Comment by olivermrbl** [comment]
We also need to increase the padding of the content in the row to follow the design:
CleanShot 2024-08-08 at 21 15 59@2x

**Comment by olivermrbl** [comment]
We're missing a subtitle on the table:

**Currently:**
CleanShot 2024-08-08 at 21 19 30@2x

**Design:**
CleanShot 2024-08-08 at 21 20 03@2x

File: packages/admin-next/dashboard/src/providers/keybind-provider/hooks.tsx [hunk 2]
```diff
@@ -264,6 +264,14 @@ export const useGlobalShortcuts = () => {
       type: "settingShortcut",
       to: "/settings/locations",
     },
+    {
+      keys: {
+        Mac: ["G", ",", "M"],
```
Comment (olivermrbl) [inline packages/admin-next/dashboard/src/providers/keybind-provider/hooks.tsx:269]: question: Would this be more fitting? Typically, the keyboard shortcut is coupled with the resource name
```suggestion
        Mac: ["G", ",", "R"],
```

**Review by olivermrbl — APPROVED** [review]
Nice, LGTM

**Pull request merged**